packages/block-editor-tools/src/hooks/use-current-theme/index.js

```diff
@@ -0,0 +1,15 @@
+import { store } from '@wordpress/core-data';
```

Comment:
Can you convert this to TS?

Reply:
I'll leave this task to https://github.com/alleyinteractive/alley-scripts/issues/106